platform/security/encryption/platform-security-encryption-api/src/main/java/ddf/security/encryption/EncryptionService.java

```diff
@@ -27,6 +28,16 @@
    */
   String decryptValue(String wrappedEncryptedValue);
 
+  /**
+   * Encrypts a plaintext string and wraps the encrypted string in a leading "ENC(" and trailing
+   * ")".
+   *
+   * @param unwrappedPlaintext a plaintext string to be encrypted.
+   * @return encrypted text wrapped in a leading "ENC(" and trailing ")".
+   */
+  @Nullable
+  String encryptValue(String unwrappedPlaintext);
```

Comment:
✏️ `unwrappedPlaintext` should also be annotated with `@Nullable`

Reply:
added